### PR TITLE
Add Converseen

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/Faster3ck/Converseen/issues/103
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/Faster3ck/Converseen/pull/102 https://github.com/Faster3ck/Converseen/pull/45

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.fasterland.converseen

File: net.fasterland.converseen.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+          "name": "imagemagick",
```
Comment (hfiguiere) [inline net.fasterland.converseen.json:18]: do you really need this?

File: net.fasterland.converseen.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Faster3ck/Converseen.git",
+                    "tag": "v0.9.9.5",
+                    "commit": "856cef30d3b0ef7d7582db0dc569a248fceb118e"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ "sed -i 's#/usr/share/metainfo/#${CMAKE_INSTALL_PREFIX}/share/metainfo/#' CMakeLists.txt" ]
```
Comment (hfiguiere) [inline net.fasterland.converseen.json:43]: make sure you submit a patch upstream for that.

File: net.fasterland.converseen.json (hunk 1 above)
Comment (Mailaender) [reply net.fasterland.converseen.json:18]: Yes, this whole application is basically a wrapper around `Magick++`.

File: net.fasterland.converseen.json (hunk 2 above)
Comment (Mailaender) [reply net.fasterland.converseen.json:43]: See https://github.com/Faster3ck/Converseen/pull/45 which was ignored and now can't be applied.

File: net.fasterland.converseen.json (hunk 1 above)
Comment (hfiguiere) [reply net.fasterland.converseen.json:18]: the binaries or the lib? because if it never uses the binaries, they should be in the cleanup.

Comment (hfiguiere) [reply net.fasterland.converseen.json:18]: it seems to require `convert`
also need to cleanup  `/share/doc`, `/include`, and maybe the static libs?

Comment (Mailaender) [reply net.fasterland.converseen.json:18]: > the binaries or the lib?
I guess both then. Added the cleanup rule for the rest.

**Comment by barthalion** [comment]
bot, build net.fasterland.converseen

**Comment by nedrichards** [comment]
bot, build net.fasterland.converseen

File: net.fasterland.converseen.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "id": "net.fasterland.converseen",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "converseen",
+    "rename-desktop-file": "converseen.desktop",
+    "rename-appdata-file": "converseen.appdata.xml",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
```
Comment (nedrichards) [inline net.fasterland.converseen.json:11]: What in particular does this need in the filesystem that it doesn't get from the portal?

Comment (Mailaender) [reply net.fasterland.converseen.json]: This is indeed not required.

**Pull request closed without merging**